source/blog/2015-04-04-ember-1-11-1-released.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Ember.js 1.11.1 Released
+author: Robert Jackson
+tags: Releases, Recent Posts
+---
+
+Today we are releasing Ember.js 1.11.1, a patch-level release of Ember that
+fixes several regressions introduced in the 1.11.1 release.
+
+### CollectionView with `attributeBindings`
+
+During the 1.11.0 cycle much effort was put into making sure that `attributeBindings` used
+the same logic that template bound attributes use.  Namely, that `attributeBindings` use
+HTMLBars' attribute helpers instead of jQuery's.
+
+Unfortunately, this work lead to an error when `CollectionView` instances were used with
+`attributeBindings` specified.  This regression is fixed in 1.11.1.
+
+### Invalid URLs
+
+Refactoring to the router initialization process lead to a bug that caused the browsers URL
+to become out of sync with the correct URL when the back button was pressed while using 
+`HistoryLocation`.
+
+This regression is fixed in 1.11.1.
+
+### Handlebars Helper Issues
+
+A couple regressions were fixed for Handlebars helpers with 1.11.1:
+
+* The `{{else}}` template was not being passed into Handlebars helpers since 1.10.0. Therefore
+calling `options.inverse` was impossible.
```

Comment:
`The \`{{else}}\` template has not being passed into Handlebars helpers since 1.9.0`